aldryn_client/localdev/main.py

```diff
@@ -239,7 +239,9 @@ def upload_media(client):
 
     click.secho('Creating archive of local media folder', fg='green')
     with tarfile.open(archive_path, mode='w:gz') as tar:
-        tar.add(os.path.join(project_home, 'data/media'), arcname='media')
+        media_dir = os.path.join(project_home, 'data/media')
```

Comment:
(Just a hint). Maybe it's not *that* important. But if you're already using `os.path.join` you could make it really platform dependent: `os.path.join(project_home, 'data', 'media')`